R/pkgs/report_generation/man/boxplot_by_timeperiod.Rd

```diff
@@ -0,0 +1,28 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
Similarly for report_generation (and i'm assuming covidcommon too)

Reply:
removed